lib/ansible/modules/cloud/cloudscale/cloudscale_floating_ip.py

```diff
@@ -137,6 +154,11 @@
   returned: success
   type: str
   sample: 185.98.122.176
+region:
+  description: The region of the floating IP.
+  returned: success when state == present
+  type: dict
+  sample: {'slug': 'lpg'}
```

Comment:
please add the `version_added: '2.10'` here as well

Reply:
Sure, sorry for not providing that up front.